src/deadline/client/cli/_groups/asset_group.py

```diff
@@ -155,3 +260,146 @@ def asset_download(**args):
     Downloads input manifest of previously submitted job.
     """
     click.echo("download complete")
+
+
+def read_local_manifest(manifest: str) -> BaseAssetManifest:
+    """
+    Read manifets specified by filepath to manifest folder, returns BaseAssetManifest Object
+    """
+    for filename in os.listdir(manifest):
+        if filename.endswith("_input"):
+            # decode text file string into BaseAssetManifest
+            filepath = os.path.join(manifest, filename)
+            with open(filepath, "r") as input_file:
+                manifest_data_str = input_file.read()
+                asset_manifest = decode_manifest(manifest_data_str)
+
+                return asset_manifest
+
+    raise NonValidInputError(
+        f"Specified manifests directory {manifest} does not contain manifest file. "
+    )
```

Comment:
I'm mainly doing since the manifest folder will also include another file for S3 mapping. I think we talked about how there might be more than one manifest file for windows, but thats a small edge case we will forgo for now ?

Reply:
should I add just a single _input check for now ?

Reply:
spoken offline, will add checks for single manifest file